src/datasources/workspace/components/WorkspaceQueryEditor.tsx

```diff
@@ -1,33 +1,16 @@
-import React, { ChangeEvent } from 'react';
-import { Input } from '@grafana/ui';
+import React, { useEffect } from 'react';
+import { WorkspaceQuery } from '../types';
 import { QueryEditorProps } from '@grafana/data';
-import { InlineField } from 'core/components/InlineField';
 import { WorkspaceDataSource } from '../WorkspaceDataSource';
-import { WorkspaceQuery } from '../types';
 
 type Props = QueryEditorProps<WorkspaceDataSource, WorkspaceQuery>;
 
-export function WorkspaceQueryEditor({ query, onChange, onRunQuery }: Props) {
-  const onQueryTextChange = (event: ChangeEvent<HTMLInputElement>) => {
-    onChange({ ...query, queryText: event.target.value });
-  };
-
-  const onConstantChange = (event: ChangeEvent<HTMLInputElement>) => {
-    onChange({ ...query, constant: parseFloat(event.target.value) });
-    // executes the query
+export function WorkspaceQueryEditor({ onRunQuery }: Props) {
+  useEffect(() => {
     onRunQuery();
-  };
-
-  const { queryText, constant } = query;
+  }, [onRunQuery]);
 
   return (
-    <>
-      <InlineField label="Constant">
-        <Input onChange={onConstantChange} value={constant} width={8} type="number" step="0.1" />
-      </InlineField>
-      <InlineField label="Query Text" labelWidth={16} tooltip="Not used yet">
-        <Input onChange={onQueryTextChange} value={queryText || ''} />
-      </InlineField>
-    </>
+    <span>This data source does not include a query editor - it simply returns all of the SystemLink Workspaces.</span>
```

Comment:
@MelissaHilliard - If you have a few could you review this string? It shows up in a panel after selecting this data source.

![image](https://github.com/ni/systemlink-grafana-plugins/assets/25089669/cc1fe904-c407-4280-9ce8-8153ceb41696)

Reply:
```suggestion
    <span>This data source returns all SystemLink workspaces and does not include a query editor.</span>
```